stumpwm.texi.in

```diff
@@ -436,13 +436,12 @@ These frames can be further split, creating nested boxes.
 
 Technically speaking, frames live within a ``frame tree''. When you
 split a frame, the command actually creates @emph{two} new frames
-side-by-side within the original parent frame. This makes no practical
-difference, unless you use the `sibling' command, which will move to
-the other child frame within the parent frame.
+side-by-side within the original parent frame. This can make a difference when
+you use the `resize' or `sibling' commands.
```

Comment:
I've been using stump for a decade and I had no idea about the sibling command!

I would omit the bit about `resize` here: I'm not 100% sure I see the connection or the way that frame trees impact resizing. I have a vague sense, but this doesn't clear things up for me.

Reply:
That's kind of the point; the frame tree does impact resizing. Consider these two operations:
A. only, hsplit, vsplit, move-focus right, vsplit, move-focus left
B. only, vsplit, hsplit, move-focus down, hsplit, move-focus up

In both cases, you end up with 2x2 frames of equal size, with the focus on the top left. However,
1) If you increase the width of the current frame, that also affects the width of the frame below in case A but not in B.
2) Analogously if you increase the height, it affects the frame to the right in case B but not in A.

Not sure how to increase clarity without significantly increasing the word count. Maybe just remove the idea that it doesn't make a difference, because it does. `remove` is another command that is affected.

Reply:
Ah I see what you mean, and yeah, I'm aware of the behavior that you describe, and I think it makes sense to me now, so it's sort of "obvious" but I see how it isn't necessarily so.

I think your change is definitely an improvement, though I might just say "`sibling` and other commands that manipulate split frames" or some such.

Reply:
Yeah, it's kind of the expected implication of the easiest implementation. But as an example of how it's not obvious, I'm migrating from ratpoison which does not have this behaviour.

See `e7439e0`.

Reply:
sounds good to me!